### DISCUSSION TITLE
Beggar (Neutral)

### DISCUSSION THREAD

**Discussion by Alf21** [Alf21/TTT2#777] [Role Ideas]
## The Beggar

**Description** He is a person in need and happy to receive any gift he can get.

**Win condition and alignment** You may change your role to that of another player and take on their win condition and alignment.

**Gameplay and Abilities**
- The Beggar starts without a team and cannot win by roundstart.
- If the beggar picks up a dropped Shopitem he will take on the role of the one dropping it to him.
- Anyone with a shop will be able to see who the Beggar is.

**Plays well with**
- Survivalist
- Serial Killer
- Necromancer
- Jackal

Idea by: @LunexSan

**Comment by TaintedEnergy** [comment]
This role has been created: https://steamcommunity.com/sharedfiles/filedetails/?id=2605752081
